modules/core/src/test/java/org/apache/ignite/internal/processors/service/ServiceDeploymentProcessIdSelfTest.java

```diff
@@ -83,13 +87,17 @@ public ServiceDeploymentProcessIdSelfTest(IgniteBiTuple<DiscoveryEvent, Affinity
         customEvt.eventNode(node);
 
         return Arrays.asList(new Object[][] {
-            {new IgniteBiTuple<>(customEvt, new AffinityTopologyVersion(ThreadLocalRandom.current().nextLong()))},
-            {new IgniteBiTuple<>(evt, new AffinityTopologyVersion(ThreadLocalRandom.current().nextLong()))}});
+            {customEvt.getClass().getSimpleName(), new IgniteBiTuple<>(customEvt, new AffinityTopologyVersion(ThreadLocalRandom.current().nextLong()))},
+            {evt.getClass().getSimpleName(), new IgniteBiTuple<>(evt, new AffinityTopologyVersion(ThreadLocalRandom.current().nextLong()))}});
     }
 
     /** */
     @Test
     public void topologyVersion() {
+        System.out.println("test with event type: " + testLabel);
```

Comment:
Can we please log it instead of println?

Reply:
fixed